packages/vuetify/src/components/VApp/VApp.ts

```diff
@@ -34,6 +34,12 @@ export default mixins(
     },
   },
 
+  beforeCreate () {
+    if (process.env.NODE_ENV === 'development' && (!this.$vuetify || this.$vuetify === this.$root)) {
```

Comment:
> `this.$vuetify === this.$root`

When does that happen?

Reply:
When vuetify is not bootstrapped properly. 

*dev/index.js*

```js
import Vue from 'vue'
import App from './App'
import Vuetify from 'vuetify'
// import * as locales from '../src/locale'
import router from './router'
import '@mdi/font/css/materialdesignicons.css'

Vue.config.performance = true

Vue.use(Vuetify)

// const vuetify = new Vuetify({
//   lang: {
//     locales,
//   },
// })

const vm = new Vue({
  data: () => ({ isLoaded: document.readyState === 'complete' }),
  router,
  render (h) {
    return this.isLoaded ? h(App) : undefined
  },
}).$mount('#app')

// Prevent layout jump while waiting for styles
vm.isLoaded || window.addEventListener('load', () => {
  vm.isLoaded = true
})
```

*VApp.ts*
```ts
  render (h) {
    console.log(this.$vuetify === this.$root)
```

![image](https://user-images.githubusercontent.com/15625235/61847874-832ec880-aeac-11e9-846c-83a6bbceb7d4.png)

Don't ask me why. Although maybe it shouldn't happen and there's another issue with it

Reply:
Yeah that looks like a bug. This check could be in src/install.ts:49 with the logic there fixed to return undefined